boards/holybro/can-gps-v1/src/init.c

```diff
@@ -135,6 +135,8 @@ stm32_boardinitialize(void)
 
 __EXPORT int board_app_initialize(uintptr_t arg)
 {
+	VDD_3V3_SENSORS_EN(true);
```

Comment:
If you're going to do it manually should PC15 be taken away from SPI bus power control?

https://github.com/PX4/PX4-Autopilot/blob/8d46772444e19a7b29534f435b97187390221681/boards/holybro/can-gps-v1/src/spi.cpp#L41

Reply:
It is a good point. 

Initial state is Off.
VDD_3V3_SENSORS_EN(true); sets it on
stm32_spiinitialize will turn it off and then on again.

![image](https://user-images.githubusercontent.com/1945821/134418728-1e0f7891-6647-4e8b-bd29-e2ed81971e48.png)

But it us not needed. The time to move the rail from 3.3V to 3.13V (-5%) is about 100 uS. 

The time the pin is then turned off and on again in stm32_spiinitialize  is about 2.2 uS. The rail will not move.